src/backend/access/table/tableam.c

```diff
@@ -763,4 +763,5 @@ Oid
 get_tde_table_am_oid(void)
 {
 	return get_table_am_oid("tde_heap", false);
-}
\ No newline at end of file
+}
+
```

Comment:
Why is this file changed in this commit? It should probably be changed in its own commit.